dwitter/templates/404.html

```diff
@@ -0,0 +1,7 @@
+{% extends 'base.html' %}
+
+{% block content %}
+
+    <h2 class=header-title>This dweet doesn't exist.</h2>
```

Comment:
You'd get a 404 for other cases than missing dweets. Maybe change to "this page doesn't exist"